src/stdlib.jl

```diff
@@ -1,25 +1,25 @@
 import Base.CoreLogging:
     AbstractLogger,
+    LogLevel,
     handle_message,
     min_enabled_level,
     shouldlog,
-    global_logger,
-    Debug
+    global_logger
 
-const LEVEL_MAP = Dict{AbstractString}
 struct CoreLogger <: AbstractLogger
+    min_level::LogLevel
```

Comment:
Should be a breaking release as now CoreLoggers need to be defined with a log level

Reply:
While we are at it can we do a better name than `CoreLogger`?

Reply:
> Should be a breaking release as now CoreLoggers need to be defined with a log level

CoreLoggers is intentionally unexported.

> While we are at it can we do a better name than CoreLogger?

I'm open to suggestions, but nobody is supposed to interact with this type directly. They should just be calling `substitute!([level])`.

Reply:
StdLibLogger?

Reply:
> CoreLoggers is intentionally unexported,

Right, I just figured it would be exported.

Reply:
FWIW, the reason I intentionally didn't export `CoreLogger` is because loggers in base are closer to handlers in memento, so I feel like exporting both types of loggers would add to further confusion. Sure, I'm fine with renaming it to `StdLibLogger` or even `LoggingLogger` if folks want.

Reply:
I think `CoreLogger` may be the right name. Most of the logging in Julia exists in a module called `CoreLogging` and the stdlib just implements a `ConsoleLogger`.

Reply:
Okay, thumbs up or down the following options below. NOTE: I wish github had a builtin poll feature.

Reply:
CoreLogger

Reply:
StdLibLogger

Reply:
LoggingLogger

Reply:
Conceptually, I think of the logging definitions that happens in Base.CoreLogging as a unfortunate implementation detail and that it would be in the Logging StdLib except that Base can't have dependencies.

Further Base is a stdlib really.

Main problem with Core is that it sounds like it refers to the Core stdlib,
So BaseLogger would be preferable.

Reply:
BaseLogger

Reply:
Okay, I guess BaseLogger wins... if 😆 < 👍